src/connectedk8s/azext_connectedk8s/_help.py

```diff
@@ -79,3 +79,39 @@
     - name: Show the details for a connected kubernetes cluster
       text: az connectedk8s show -g resourceGroupName -n connectedClusterName
 """
+
+helps['connectedk8s proxy'] = """
+  type: command
+  short-summary: Get access to a connected kubernetes cluster.
+  examples:
+  - name: Get access to a connected kubernetes cluster.
+    text: az connectedk8s proxy -n clusterName -g resourceGroupName
+  - name: Get access to a connected kubernetes cluster with custom 'client proxy port' and 'api server port'
+    text: az connectedk8s proxy -n clusterName -g resourceGroupName --api-server-port portValue --client-proxy-port portValue
```

Comment:
>client [](start = 98, length = 6)

-client-proxy-port should just be port. Keeping it consistent with kubectl proxy [https://kubernetes.io/docs/reference/generated/kubectl/kubectl-commands#proxy](https://kubernetes.io/docs/reference/generated/kubectl/kubectl-commands#proxy)

Reply:
Removed exposure of 'client proxy port' to user after our discussion. Also updated the CLIError if 'client proxy port' is in use and also added telemetry to record this exception.